### DISCUSSION TITLE
Icon Colors

### DISCUSSION THREAD

**Discussion by MrDragon420** [Jyers/Marlin#34] [Ideas]
Can we get some new color on the icons? i've seen a firmwhere that someone fliped it to the side, he has nice color icons, this would help with the low viewing angle on this...

**Comment by Jyers** [comment]
I haven't yet looked into modifying the screen icons as that involves flashing the screen, but I'm certainly going to look into it!

**Reply by kyraxyg** [reply]
https://github.com/b-pub/dwin-ico-tools

might be a good starting point. You can also change the baud rate of LCD by hex editing the config file. I am already using 230400 as baud rate for USB as well as LCD

**Reply by Jyers** [reply]
I've used that dwin tool before and it seems to work well. Thanks for the note on the baud rate, can't imagine how much benefit that would have, but certainly good to know!

**Reply by Benetos** [reply]
It would be fun to modify them but there is value in keeping the screen from needing to be flashed (esspecially around people new to firmware updates). So it might be better to have a seperate versio ndown the road but we would need to have the icons and colors made to include in the screen firmware flash.

**Reply by Jyers** [reply]
My main idea would be a kind of theme system. Where the icon configuration is kept the same but the icons themselves are changed. That way you wouldn't need to flash the screen, but if you wanted a different look you could

**Reply by DeNohvo** [reply]
I think the biggest impact would be if we can fix the viewing angles by maybe making the background a different colout?
